src/plugins/data/public/search/search_interceptor.ts

```diff
@@ -106,12 +114,13 @@ export class SearchInterceptor {
         return new EsError(e);
       }
     } else {
-      return e;
+      return new Error(e.message);
     }
   }
 
   /**
    * @internal
+   * @throws `AbortError` | Object
```

Comment:
I think this `Object` in the latest revision isn't accurate comment

Reply:
Changed to `ErrorLike`